lib/devise_saml_authenticatable/version.rb

```diff
@@ -1,3 +1,3 @@
 module DeviseSamlAuthenticatable
-  VERSION = "1.9.1"
+  VERSION = "2.0.0"
```

Comment:
Is this actually a breaking change? It seems like it's backwards-compatible, which is great! In which case, we could release `1.10.0`.

Reply:
I did this as a major version bump, as the reconfiguration of the attributes-map.yml is a bit of a pain; and people will have to adjust their custom attribute resolvers slightly.